.travis.yml

```diff
@@ -22,7 +22,7 @@ matrix:
       gemfile: gemfiles/rails_4.2.gemfile
 before_install:
  # Bundler 2.x is supported for Rails > 4.2
- - ([[ "$BUNDLE_GEMFILE" != *"rails_4.2.gemfile" ]] && gem uninstall bundler -v '>= 2' && gem install bundler -v '< 2' ) || true
+ - ([[ "$BUNDLE_GEMFILE" = "$PWD/gemfiles/rails_4.2.gemfile" ]] && gem uninstall bundler -v '>= 2' && gem install bundler -v '< 2' ) || true
```

Comment:
How about `[[ "$BUNDLE_GEMFILE" =~ rails_4.2.gemfile$ ]]`

Reply:
That works for me, for some reason JRuby still doesn't like it even with this running with 4.2 so I gave up on it for the day. I can make that change the next time I get back into it.

Reply:
I applied it now. Let's see if it makes any difference. Travis is making me really tired. Builds take ages and cant install rbx. Strange. Maybe we should give a try to CircleCI?